plugins/woocommerce/src/Internal/Admin/Reviews.php

```diff
@@ -74,6 +76,98 @@ public function add_reviews_page() {
 		add_action( "load-{$this->reviews_page_hook}", [ $this, 'load_reviews_screen' ] );
 	}
 
+	/**
+	 * Determines whether the current page is the reviews page.
+	 *
+	 * @return bool
+	 */
+	public function is_reviews_page() : bool {
+		global $pagenow;
```

Comment:
Out of curiosity, any reason not to use the `$current_screen` global?

Reply:
Yeah, `$current_screen` is better here. I had followed some examples with `$pagenow` inside WC core classes and completely forgot about that one. Thanks for pointing that out. Done in https://github.com/godaddy-wordpress/woocommerce/pull/34/commits/656bece0f9374ba90dd44175cc2c54d782bdacb0.